### ISSUE TITLE
Postfix Version

### ISSUE THREAD

**Issue by spec1re** [mailwatch/MailWatch#947]
Postfix Version stays empty in **sf_version.php** because of wrong path with FreeBSD

non-working
```
    // Add test for MTA
    $mta = get_conf_var('mta');
    if (get_conf_var('MTA', true) === 'postfix') {
        echo '<br>' . "\n";
        echo 'Postfix ' . __('version11') . ' ';
        passthru("/usr/sbin/postconf -d | grep 'mail_version =' | cut -d' ' -f3");
        echo '<br>' . "\n";
    }
```
working
```
    // Add test for MTA
    $mta = get_conf_var('mta');
    if (get_conf_var('MTA', true) === 'postfix') {
        echo '<br>' . "\n";
        echo 'Postfix ' . __('version11') . ' ';
        passthru("/usr/local/sbin/postconf -d | grep 'mail_version =' | cut -d' ' -f3");
        echo '<br>' . "\n";
    }
```

**Comment by asuweb** [comment]
@spec1re this is fixed in PR #948 - can you confirm it works on your side please?

**Comment by spec1re** [comment]
Works, thanks! :)
